src/main/java/controller/Calculator.java

```diff
@@ -0,0 +1,21 @@
+package controller;
+
+import model.CalculatorType;
+
+public class Calculator {
+
+    public int calculate(String[] expression){
+
+        int result = Integer.parseInt(expression[0]);
+        CalculatorType calculatorType;
+        
+        for (int i = 2; i < expression.length; i += 2) {
+            int next_num = Integer.parseInt(expression[i]);
+
+
+            result = calculatorType.operate(result, next_num);
```

Comment:
개행도 하나의 의미를 가지게 됩니다.
```suggestion
            int next_num = Integer.parseInt(expression[i]);

            result = calculatorType.operate(result, next_num);
```
의미없는 개행은 피해주세요 :)